test/integration/targets/roles_arg_spec/test_tags.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  gather_facts: false
+  tasks:
+      - name: "Tag test #1"
+        import_role:
+            name: a
+        vars:
+            a_str: "tag test #1"
+        tags:
+            - foo
```

Comment:
I don't even think you need a task with a tag applied to it to test this. Just running any of the other playbooks with `--tags bar` should run the validation task (but no role tasks).

Reply:
true, but does not hurt either

Reply:
Yeah, I guess that's true. Doesn't hurt to be explicit here either.